### PR TITLE
wayland_1_9: remove

### PR DESCRIPTION
**Pull request by globin**
Ancient compat version, that no longer has any use
It was introduced > 3 yrs ago in 0593ad2b1613e22efa88307ee6ee4115563721fb

### CODE REVIEW

File: nixos/modules/tasks/network-interfaces-systemd.nix [hunk 1]
```diff
@@ -72,15 +72,6 @@ in
...
         enable = true;
-        networks."99-main" = (genericNetwork mkDefault) // {
-          # We keep the "broken" behaviour of applying this to all interfaces.
-          # In general we want to get rid of this workaround but there hasn't
-          # been any work on that.
-          # See the following issues for details:
-          # - https://github.com/NixOS/nixpkgs/issues/18962
-          # - https://github.com/NixOS/nixpkgs/issues/61629
-          matchConfig = mkDefault { Name = "*"; };
-        };
```
Comment (hedning) [inline nixos/modules/tasks/network-interfaces-systemd.nix]: This doesn't look related?

Comment (globin) [reply nixos/modules/tasks/network-interfaces-systemd.nix]: Fixed, thanks

**Review by jonringer — APPROVED** [review]
